slacktee.sh

```diff
@@ -245,7 +272,8 @@ function setup()
 	if [[ -z "$input_icon" ]]; then
 		input_icon=$icon
 	fi
-	read -p "Default color of the attachment. (empty string disables attachment) [$attachment]: " input_attachment
+	read -p "Default color of the attachment. \
```

Comment:
This creates unnecessary spaces between `attachment.` and `(empty`.

Here is the screenshot.
https://gyazo.com/f1fbfdb74054d6e05c3c346a53884a20